store/src/main/java/org/apache/rocketmq/store/MappedFile.java

```diff
@@ -371,18 +371,18 @@ private boolean isAbleToFlush(final int flushLeastPages) {
     }
 
     protected boolean isAbleToCommit(final int commitLeastPages) {
-        int flush = this.committedPosition.get();
+        int commit = this.committedPosition.get();
```

Comment:
I think `flush` is correct,This represents the offset of the flush without changing the variable name

Reply:
> I think `flush` is correct,This represents the offset of the flush without changing the variable name

I think so too

Reply:
Hi, thx for your review. But I believe the name "commit" would be more suitable for the usage of getting the previous committed position. Using "flush" may be confused here since this.flushedPosition.get() in isAbleToFlush() method is also named "flush" to represent getting the previous flushed position.